### DISCUSSION TITLE
Pksim-parameter identification

### DISCUSSION THREAD

**Discussion by ZML0113** [Open-Systems-Pharmacology/Forum#1367] [Models] [answered]
When I tried doing a parameter identification, what should I add to the observed data? And the error was as follows:
<img width="810" alt="1666771586980" src="https://user-images.githubusercontent.com/81542635/197970077-a78795c8-82c5-43f2-a7fd-34a815fbccce.png">

**Comment by madiparrot** [comment]
I also have this same error! Please let me know if you figure it out

**Reply by madiparrot** [reply]
I did notice though that you have not added observed data? You can add each parameter separately and create a range for possible values

**Comment by prvmalik** [comment, marked as answer]
You will need to insert observed data onto your simulation by dragging and dropping. Then you will be able to select it in the "Observed Data" dropdown of the parameter identification tool to be matched to one of the outputs.

**Reply by madiparrot** [reply]
Directly into the individual simulation?

**Reply by prvmalik** [reply]
Correct. then it can be selected in the parameter identification tool.
